.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -72,7 +77,8 @@ phases:
     steps:
       - bash: |
           echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
-          echo '##vso[task.setvariable variable=UBUNTU_SKU]18.04'
+          echo '##vso[task.setvariable variable=OS_DISTRO_FAMILY]Ubuntu'
```

Comment:
In our API we plan to have OS SKU with value Ubuntu or Mariner, can we keep it consistent and call it OS_SKU here?